src/ui/menu.ts

```diff
@@ -75,7 +75,12 @@ async function menu(_ui: UI) {
   let useAutofill = (localStorage.autofill === 'true');
 
   const ui: UIConfig = {
-    data: {version, zoom, useAutofill},
+    data: {
+      version,
+      zoom,
+      useAutofill,
+      newStorageLocation: localStorage.storageLocation
```

Comment:
`newStorageLocation` should have a default value. `newStorageLocation: localStorage.storageLocation || 'sync'`